proxy/src/control/discovery.rs

```diff
@@ -506,7 +597,107 @@ where T: HttpService<RequestBody = BoxBody, ResponseBody = RecvBody>,
     }
 }
 
-// ===== impl RxError =====
+// ===== impl DstLabels ====
+
+impl DstLabels {
+
+    #[inline]
```

Comment:
Why is this `#[inline]? It is quite a huge function to inline.

Reply:
Removed in aae6527fca40580a2678fa794b53931c329638b7.